docs/recipes.md

```diff
@@ -0,0 +1,58 @@
+<!--
+SPDX-FileCopyrightText: 2023 Springtime authors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+We provide several "recipes" for downloading data from various sources and
+executing a range of ML methods with it.
+Note that a "recipe" is a file with `yaml` extension.
```

Comment:
We're still missing a good explanation of what are/how to use the recipes vs the API. See other comment about rewriting the example use case.

Reply:
I added some explanations under a new title "Usage" in the readme, see my [commit](https://github.com/phenology/springtime/pull/105/commits/df6c172404194c348ad49598ec35e53755e1a381).